src/components/TestnetInfo/TestnetInfo.component.tsx

```diff
@@ -0,0 +1,29 @@
+import { Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+import { useSelector } from 'react-redux';
+import { isOnTestnetSelector } from '../../store/app/app.selectors';
+
+export const TestnetInfo = () => {
+  const isOnTestnet = useSelector(isOnTestnetSelector);
+  return isOnTestnet ? (
```

Comment:
Maybe something like that would be clearer?

```suggestion
  if (!isOnTestnet) return null
  
  return <Box...
```